index.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html>
+    <head>
+        <title>PHP Cornwall</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="PHP Cornwall Community Group">
+        <meta name="keywords" content="PHP, Wordpress, Cornwall, Web Dev">
+        <meta name="theme-color" content="#dee4ff">
+        <link rel="stylesheet" type="text/css" href="assets/css/output.css">
+        <link rel="stylesheet" type="text/css" href="assets/css/normalize.css">
```

Comment:
We should be loading normalize before our custom CSS otherwise some of it could be overwritten 😬

Reply:
Ah okay, moved `normalize` before the custom CSS now